README.md

```diff
@@ -1,72 +1,25 @@
-# A Regularized Conditional GAN for Posterior Sampling in Inverse Problems [[arXiv]](https://arxiv.org/abs/2210.13389)
+# Generative modelling for mass-mapping with fast uncertainty quantification [[arXiv]](https://arxiv.org/abs/2410.24197)
 ## Setup
 See ```docs/setup.md``` for basic environment setup instructions.
 
```

Comment:
You need to add a reference to the `comments.md` file where we explain how to install the model and some other useful stuff. There are two option, mention what you can find on that file and provide a link or merge the info of that file in the readme